timelock-server/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosResource.java

```diff
@@ -49,16 +51,32 @@ public static PaxosResource create() {
     }
 
     public static PaxosResource create(String logDirectory) {
-        return new PaxosResource(logDirectory, Maps.newHashMap(), Maps.newHashMap());
+        return new PaxosResource(logDirectory, Maps.newConcurrentMap(), Maps.newConcurrentMap());
```

Comment:
It's probably safe since we set up these maps in the run method before any request threads can hit them. But as @schlosna pointed out, probably best to make these threadsafe given that there is a public method that mutates them.

Also, I could see us using these for live-reload